README.md

```diff
@@ -2,10 +2,43 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
+The TSC is community and contributor driven and is a home for community managed 
+projects and assets. The mission of the Node.js Foundation is to enable and encourage 
```

Comment:
This is a little complicated in terms of phrasing.

The technical community driven part of the foundation is managed by the TSC. To say something is "in the Node.js Foundation" when it is a technical project managed by its contributors and community that means it is under the TSC and managed on the the technical side of the foundation.

The "incubator" is how we admit projects, but to say "the incubator is home to projects" is only part of the story because eventually projects graduate from the incubator but are still under the TSC.

But yes, this is hard to articulate well and I'm open to suggestions for better wording.